src/projectM-sdl/pmSDL.cpp

```diff
@@ -228,15 +236,107 @@ projectMSDL::projectMSDL(std::string config_file, int flags) : projectM(config_f
     isFullScreen = false;
 }
 
-void projectMSDL::init(SDL_Window *window, SDL_GLContext *_glCtx) {
+void projectMSDL::init(SDL_Window *window, SDL_GLContext *_glCtx, const bool _renderToTexture) {
     win = window;
     glCtx = _glCtx;
-    selectRandom(true);
     projectM_resetGL(width, height);
+
+    renderToTexture = _renderToTexture;
+
+    if (renderToTexture) {
```

Comment:
this code shouldn't be in pm-sdl right? should be in the library?

Reply:
I don't think so, it is left to the developer to decide where and how he will draw the projectM generated texture.